app/api/helpers/feedback.py

```diff
@@ -0,0 +1,18 @@
+import logging
+from app.models import db
+
+logger = logging.getLogger(__name__)
+
+
+def delete_feedback(feedback):
+    """
+    Delete the feedback if rating is zero
+    :param feedback: Feedback to be deleted
+    :return:
+    """
+    db.session.delete(feedback)
+    try:
+        db.session.commit()
+    except Exception as e:
+        logging.exception('DB Exception! %s' % e)
```

Comment:
Still using root logging module.

Also, DB Exception is useless. It doesn't tell where the error happened.

Also, no need to include e in the log, stacktrace is automatically printed

Reply:
Ah, missed it. Fixed it now